test/testModel.cc

```diff
@@ -430,12 +430,24 @@ SUITE(Canvas)
     {
       cairo::Surface surf(cairo_recording_surface_create(CAIRO_CONTENT_COLOR,nullptr));
       c->draw(surf.cairo());// reposition ports
-      CHECK(c->clickType(c->x(),c->y()) == ClickType::onItem);
+      CHECK(c->clickType(c->x(),c->y()) == ClickType::onItem); 
       canvas.mouseDown(c->x(),c->y());
       canvas.mouseUp(400,500);
       CHECK_EQUAL(400,c->x());
       CHECK_EQUAL(500,c->y());
     }
+    
+    TEST_FIXTURE(TestFixture,resizeItem)
+    {
+      cairo::Surface surf(cairo_recording_surface_create(CAIRO_CONTENT_COLOR,nullptr));
+      c->draw(surf.cairo());// reposition ports
+      float xc=c->x()+0.5*c->width(), yc=c->y()+0.5*c->height();      
+      CHECK(c->clickType(xc,yc) == ClickType::onResize); 
+      canvas.mouseDown(xc,yc);
+      canvas.mouseUp(800,1000);
+      CHECK_EQUAL(1062,c->x()+0.5*c->width());
+      CHECK_EQUAL(1460,c->y()+0.5*c->height());
```

Comment:
Left hand constants are brittle here. Better to calculate them - eg 800-xc == 0.5*c->width(), etc.

Reply:
Sort of fixed this... But the default item does not resize properly. Is it perhaps a Pango related issue?